TODO-rebase.sh

```diff
@@ -529,20 +529,20 @@ fi
 # 	force_rebase=t
 # fi
 
-if test -z "$rebase_root"
-then
-	case "$#" in
-	0)
-		if ! upstream_name=$(git rev-parse --symbolic-full-name \
-			--verify -q @{upstream} 2>/dev/null)
-		then
-			. git-parse-remote
-			error_on_missing_default_upstream "rebase" "rebase" \
-				"against" "git rebase $(gettext '<branch>')"
-		fi
+# if test -z "$rebase_root"
+# then
+# 	case "$#" in
+# 	0)
+# 		if ! upstream_name=$(git rev-parse --symbolic-full-name \
+# 			--verify -q @{upstream} 2>/dev/null)
+# 		then
+# 			. git-parse-remote
+# 			error_on_missing_default_upstream "rebase" "rebase" \
+# 				"against" "git rebase $(gettext '<branch>')"
+# 		fi
 
-		test "$fork_point" = auto && fork_point=t
-		;;
+# 		test "$fork_point" = auto && fork_point=t
+# 		;;
 	# *)	upstream_name="$1"
 	# 	if test "$upstream_name" = "-"
 	# 	then
```

Comment:
Please keep updates to `TODO-rebase.sh` separate from the "real" patches, at all times. It just adds too much churn having to go back and split patches again.